course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -332,14 +335,12 @@
             var membersByUserId = {};
             memberList.forEach(function(member) {
                 var memberCopy = angular.copy(member);
-                // this is a temp fix to change the display text of the role
-                // "Teaching Fellow" to "TA" a more perm solution is being
-                // discussed, but will involve talking to the schools.
+
                 if (memberCopy.role && memberCopy.role.role_name == "Teaching Fellow") {
                     memberCopy.role.role_name = "TA";
                 }
                 membersByUserId[memberCopy.user_id] =
-                    (membersByUserId[memberCopy.user_id] || []).push(memberCopy);
+                     (membersByUserId[memberCopy.user_id] || []).concat(memberCopy);
```

Comment:
I think we may be able to go back and fix an ugliness in one of the dialogs...I had to pull from the profile lookup instead of the memberships because the memberships just had a value of `'1'`.  :)